test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/HelpInfoTests.cs

```diff
@@ -250,6 +282,50 @@ public async Task Help_WithAllExtensionsRegistered_OutputFullHelpContent(string
         testHostResult.AssertOutputMatches(wildcardPattern);
     }
 
+    [ArgumentsProvider(nameof(TargetFrameworks.All), typeof(TargetFrameworks))]
+    public async Task HelpShortName2_WithAllExtensionsRegistered_OutputFullHelpContent(string tfm)
+    {
+        var testHost = TestInfrastructure.TestHost.LocateFrom(_testAssetFixture.AllExtensionsTargetAssetPath, TestAssetFixture.AllExtensionsAssetName, tfm);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--?");
```

Comment:
Same remarks here for just checking part of the help and `-?`

Reply:
yes -? works and added a test for it